test/test_viewmodel.py

```diff
@@ -0,0 +1,12 @@
+from todo_app.data.session_items import ViewModel,MyItem
+
+
+def test_status():
```

Comment:
A nice simple example, it would be good to make this a little more targeted by splitting it out into two separate tests, each with the same set up but splitting the assert statements, so that if the test were to fail you'd have a clear idea of which function to investigate